src/Signature.gs

```diff
@@ -68,12 +87,14 @@ function requestSignature(token) {
 function signDocument(token, signatureImageId) {
     var preferredHeight = 60;
 
-    /* TODO  dummy code until api ready */
-    var signatureImage = oldGetSignatureImage(signatureImageId);
-    placeSignature(signatureImage,preferredHeight);
-    return;
-    /* --- */
+    // circuit breaker
+    if(backendDissabled){
```

Comment:
Now it looks like if the back-end **is not working**, then send the request to the beck-end. Maybe it should be in opposite way.